src/status_im/extensions/ethereum.cljs

```diff
@@ -234,12 +234,39 @@
                         :on-failure on-failure}]
      (wrap-with-resolution db json-rpc-args :to execute-send-transaction))))
 
-(defn- parse-log [{:keys [address transactionHash blockHash transactionIndex topics blockNumber logIndex removed data]}]
-  (merge {:data    data
+(defn- event-topic-enc [event params]
+  (let [eventid (str event "(" (string/join "," params) ")")]
+    (abi-spec/sha3 eventid)))
+
+; Return a vector without elements of Y
+(defn diff-vectors [X Y]
+  (reduce #(remove (fn [x] (= x %2)) %1) X Y))
```

Comment:
There are probably simpler ways to do that, like using `remove` or switching to ``clojure.set` if values in the vector are unique.

Reply:
The problem here is that the vector hasn't unique elements, so I can't use a Clojure set with `remove` function and I didn't find a more simple/easy/more readable way to do it. Any clue or suggestion?